### PR TITLE
Add VPC and two EC2 with security group

### PR DESCRIPTION
**Pull request by maurodandrea**
Added the following resources:
- single VPC with two different network, public (but not have a public access) and private.
- VPC EndPoint for S3/BynamoDB
- two EC2 istances, first EC2 in public network and second EC2 in private network
- two security group that allow ping from EC2_one to EC2_two and the connection via Session Manager to EC2_one

### CODE REVIEW

File: infra/ec2.tf [hunk 1]
```diff
@@ -0,0 +1,67 @@
+module "ec2_instance_dataexfiltration" {
+  source = "terraform-aws-modules/ec2-instance/aws"
```
Comment (pasqualedevita) [inline infra/ec2.tf]: please add version module

File: infra/vpc.tf [hunk 2]
```diff
@@ -0,0 +1,73 @@
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+module "vpc_dataexfiltration" {
+  source                       = "terraform-aws-modules/vpc/aws"
+  version                      = "5.1.2"
+  name                         = var.prefix
+  cidr                         = "10.0.0.0/24"
```
Comment (pasqualedevita) [inline infra/vpc.tf]: can we use larger cidr like `10.0.0.0/16`? in this way we can set subnets to /24

File: infra/ec2.tf [hunk 3]
```diff
@@ -0,0 +1,67 @@
+module "ec2_instance_dataexfiltration" {
```
Comment (pasqualedevita) [inline infra/ec2.tf]:
```suggestion
module "ec2_instance_dataexfiltration_one" {
```

**Review by pasqualedevita — CHANGES_REQUESTED** [review]
only a few changes

File: infra/vpc.tf (hunk 2 above)
Comment (maurodandrea) [reply infra/vpc.tf]: change present in the new commit

File: infra/ec2.tf (hunk 1 above)
Comment (maurodandrea) [reply infra/ec2.tf]: change present in the new commit

**Review by pasqualedevita — APPROVED** [review]

**Review by pasqualedevita — APPROVED** [review]

**Comment by pasqualedevita** [comment]
thank you @maurodandrea

**Pull request merged**